bookkeeper-server/src/main/java/org/apache/bookkeeper/client/EnsemblePlacementPolicy.java

```diff
@@ -251,6 +251,9 @@ EnsemblePlacementPolicy initialize(ClientConfiguration conf,
      * <p>{@code customMetadata} is the same user defined data that user provides
      * when {@link BookKeeper#createLedger(int, int, int, BookKeeper.DigestType, byte[], Map)}.
      *
+     * <p>If 'enforceMinNumRacksPerWriteQuorum' config is enabled then the bookies belonging to default
+     * faultzone (rack) will be excluded while selecting bookies. This is to enable strict placementpolicy.
```

Comment:
Not sure if the "This is to enable strict placementpolicy." is needed. Rest of the wording gives enough context.

Reply:
will remove